### DISCUSSION TITLE
Send update on every power value change

### DISCUSSION THREAD

**Discussion by LeeWiu** [arendst/Tasmota#14933] [Q&A] [answered]
Hi, I flashed tasmota on BlitzWolf SHP6 15A plug and it’s working great but it sends power value only every 300 seconds which is default for TelePeriod, even if in this period value has changed 10 times. How can I setup tasmota device to transmit everytime power value changed? Using Home Assistant.

**Comment by sfromis** [comment, marked as answer]
Use [PowerDelta](https://tasmota.github.io/docs/Commands/#powerdelta). As a starting point, I like a value of 110 for a 10W change to trigger a new payload.
